python-checks/src/main/java/org/sonar/python/checks/IdentityComparisonWithCachedTypesCheck.java

```diff
@@ -49,12 +52,40 @@ public class IdentityComparisonWithCachedTypesCheck extends PythonSubscriptionCh
   public static final String IS_QUICK_FIX_MESSAGE = "Replace with \"==\"";
   public static final String IS_NOT_QUICK_FIX_MESSAGE = "Replace with \"!=\"";
 
+  /**
+   * Fully qualified names of constructors and functions that would are guaranteed to create fresh objects with
```

Comment:
I know you didn't add this, but there's a typo here (`that would are guaranteed` -> `that are guaranteed`). Can you take the opportunity to fix it?